pkg/network/node/ovs/ovs.go

```diff
@@ -214,6 +214,9 @@ func (ovsif *ovsExec) AddBridge(properties ...string) error {
 		args = append(args, "--", "set", "Bridge", ovsif.bridge)
 		args = append(args, properties...)
 	}
+	if mayExist {
+		args = append([]string{"--may-exist"}, args...)
+	}
```

Comment:
As commented before, I'd just add this before along the another args and drop the argument ;)

Reply:
ack